### ISSUE TITLE
Rethink the note system

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/Global-signbank#478]
Julie:

> Speaking of notes, now if I don’t add a number but I’ve added text, if I commit the text without adding a number, I lose the note. Annoying… and I would prefer to get rid of the index but keep “type”. If have both, fine but want “type” back. 

Why do users have worry about note indices at all?

**Comment by ocrasborn** [comment]
The background is that in the Auslan Signbank, notes were used (also) for definitions of signs. To order these, the index number was added. Understandable. We now use notes for a variety of things, and manually maintaining the index isn't really on our minds. But perhaps it will be later, so rather than deleting the index, I would suggest (for both NGT and ASL), that we set index to 1 by default when creating a new note. That way the user doesn't have to do anything, unless s/he specifically wants something to appear at the top or bottom. Or perhaps default of 3, so that it's easier to move things up to index 1 or 2 without having to change all the notes with the default number?
